app/common/actions/tabActions.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const dispatchAction = require('../../common/dispatcher/dispatchAction')
+
+const tabActions = {
+  // TODO(bridiver) - this is an action anti-pattern and the code that uses it should be refactored
+  // to be declarative
+  reload: function tabActionsReload (tabId, ignoreCache) {
+    dispatchAction(tabActions.reload.name, {
```

Comment:
I like that it's split out into a different action file like we previously discussed; however, I think that we'll eventually hit an unintentional bug because of not namespacing on the object or module and using just the function name.  I won't block the review on that but I think we should change it in the near future since it'll be prone to collisions.